src/results.hpp

```diff
@@ -295,6 +301,28 @@ size_t Results::index_of(Context& ctx, T value)
 {
     return dispatch([&](auto t) { return this->index_of(ctx.template unbox<std::decay_t<decltype(*t)>>(value)); });
 }
+
+template<typename T>
+auto Results::list_get_as(size_t ndx) const
```

Comment:
There doesn't appear to be any reason for these to be defined in the header.

Reply:
Moved to .cpp